library/patient.inc

```diff
@@ -1044,61 +1045,36 @@ function pdValueOrNull($key, $value)
     }
 }
 
-// Create or update patient data from an array.
-//
+/**
+ * Create or update patient data from an array.
+ *
+ * This is a wrapper function for the PatientService which is now the single point
+ * of patient creation and update.
+ *
+ * If successful, returns the pid of the patient
+ *
+ * @param $pid
+ * @param $new
+ * @param false $create
+ * @return mixed
+ */
 function updatePatientData($pid, $new, $create = false)
 {
-  /*******************************************************************
-    $real = getPatientData($pid);
-    foreach ($new as $key => $value)
-        $real[$key] = $value;
-    $real['date'] = "'+NOW()+'";
-    $real['id'] = "";
-    $sql = "insert into patient_data set ";
-    foreach ($real as $key => $value)
-        $sql .= $key." = '$value', ";
-    $sql = substr($sql, 0, -2);
-    return sqlInsert($sql);
-  *******************************************************************/
-
-  // The above was broken, though seems intent to insert a new patient_data
-  // row for each update.  A good idea, but nothing is doing that yet so
-  // the code below does not yet attempt it.
-
-    if ($create) {
-        $sql = "INSERT INTO patient_data SET pid = '" . add_escape_custom($pid) . "', date = NOW()";
-        foreach ($new as $key => $value) {
-            if ($key == 'id') {
-                continue;
-            }
-
-            $sql .= ", `$key` = " . pdValueOrNull($key, $value);
-        }
-
-        // add a uuid for the new patient
-        $sql .= ", `uuid` ='" . add_escape_custom((new UuidRegistry(['table_name' => 'patient_data']))->createUuid()) . "'";
-
-        $db_id = sqlInsert($sql);
+    // Create instance of patient service
+    $patientService = new PatientService();
+    if ($create === true ||
+        $pid === null) {
```

Comment:
@mdsupport 
sounds like actions wants this:
 if (
$create === true ||
        $pid === null
) {
or this:
 if ($create === true || $pid === null) {

actions won't auto fix. running the cs phar from command line will which is what many of us do before submitting PRs.
php phpcbf.phar -n --standard=ci/phpcs.xml library/patient.inc

a little too picky for my taste.